selfdrive/test/process_replay/test_processes.py

```diff
@@ -180,17 +180,16 @@ def format_diff(results, ref_commit):
     assert len(untested) == 0, f"Cars missing routes: {str(untested)}"
 
   with concurrent.futures.ProcessPoolExecutor(max_workers=args.jobs) as pool:
-    if not args.upload_only:
-      lreaders: Any = {}
-      p1 = pool.map(get_logreader, [seg for car, seg in segments])
-      for (segment, lr) in tqdm(p1, desc="Getting Logs", total=len(segments)):
-        lreaders[segment] = lr
-
     pool_args: Any = []
     for car_brand, segment in segments:
       if (len(args.whitelist_cars) and car_brand.upper() not in args.whitelist_cars) or \
          (not len(args.whitelist_cars) and car_brand.upper() in args.blacklist_cars):
         continue
+      if not args.upload_only:
```

Comment:
I agree we should only download logs we're testing, but doesn't this just make this single-threaded now?

Reply:
yea, it was quick and dirty. 
updated it now. Hopefully there's also a way to check if the file is already downloaded?